src/pages/login/index.tsx

```diff
@@ -18,6 +19,9 @@ export const LoginPage = () => {
       window.localStorage.setItem("auth-token", data.login.token);
       window.location.href = "/home";
     },
+    onError: (error) => {
+      return error;
+    },
```

Comment:
```suggestion
    onError: (error) => error
```
ah, e esse erro não tá sendo usado né? digo, em nenhum lugar está sendo usado o valor que vc está retornando no onError 🤔

Reply:
Obrigado Beto!

Estou utilizando ele aqui:
Linha 75 -77
`<div className="login-error failed-login">{error ? error.message : ""}</div>`